www/CQMContext.js

```diff
@@ -311,4 +323,94 @@ CQMContext.prototype._getEntitlementInfo = function(entityNames, successCallback
     exec(successCallback, fail, "CQMContext", "getEntitlementInfo", [entityNames]);
 };
 
+/**
+ * Get status for an Entity such as isSavable, progressPct, progressState. (Current only supports Collection)
```

Comment:
progressPct doesn't match up with the native code in https://git.corp.adobe.com/dps/viewer-android/pull/1017 which uses downloadPercent.  The latter seems more descriptive.